filesystem/src/main/java/org/frankframework/filesystem/LocalFileSystem.java

```diff
@@ -123,6 +123,25 @@ public OutputStream createFile(Path f) throws IOException {
 		return Files.newOutputStream(f);
 	}
 
+	@Override
+	public void createFile(Path file, InputStream contents, Map<String, String> customFileAttributes) throws FileSystemException, IOException {
```

Comment:
Relevant for the CustomFileAttributes - for the localFileSystem we need to create the file first and update it with the attributes later